build.gradle.kts

```diff
@@ -516,7 +516,6 @@ tasks.register("python38PostCommit") {
 tasks.register("python39PostCommit") {
   dependsOn(":sdks:python:test-suites:dataflow:py39:postCommitIT")
   dependsOn(":sdks:python:test-suites:direct:py39:postCommitIT")
-  dependsOn(":sdks:python:test-suites:direct:py39:hdfsIntegrationTest")
```

Comment:
Changed to run hdfsIntegrationTest only in lowest and highest Python version as the functionality didn't change for long but has a history of broken due to unrelated change. If it is broken at least there are green signal for Python 3.9 / 3.10 so does not undermine other breakage signal